app/styles/app.scss

```diff
@@ -340,6 +340,26 @@ ul.preprints-block-list {
     padding-left: 0;
 }
 
+#sortBy {
```

Comment:
Why 291px? Pixels should be the last resort. It doesn't quite look right on the iPhone 5 setting:

<img width="367" alt="screen shot 2017-03-27 at 15 29 16" src="https://cloud.githubusercontent.com/assets/3374510/24374354/88f302ce-1302-11e7-9b2d-ccd8db14d928.png">